.gitignore

```diff
@@ -18,3 +19,4 @@ vendor
 temp
 tmp
 TODO.md
+yarn.lock
```

Comment:
it should not be ignored. it should exists in the repositories. https://github.com/yarnpkg/yarn/issues/963